install/helm/gloo/templates/43-gatewayparameters.yaml

```diff
@@ -24,13 +24,18 @@ spec:
     deployment:
       replicas: {{ $replicas }}
 {{- $serviceType := "LoadBalancer" -}}
+{{- $externalTrafficPolicy := "Cluster" -}}
 {{- if $gg.service -}}
 {{- if $gg.service.type -}}
 {{- $serviceType = $gg.service.type -}}
 {{- end -}}{{/* if $gg.service.type */}}
+{{- if $gg.service.externalTrafficPolicy -}}
+{{- $externalTrafficPolicy = $gg.service.externalTrafficPolicy -}}
+{{- end -}}{{/* if $gg.service.externalTrafficPolicy */}}
```

Comment:
I have little experience with Helm templating but why not have these lines right after setting the default value of `$externalTrafficPolicy` on line 27?